components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ExternalizedConsentPageConfig.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.common.model;
+
+import org.apache.axiom.om.OMElement;
+
+import java.io.Serializable;
+import java.util.Iterator;
+
+import javax.xml.bind.annotation.XmlElement;
+
+/**
+ * Representation of an ExternalizedConsentPage Config.
+ */
+public class ExternalizedConsentPageConfig implements Serializable {
+
+    private static final long serialVersionUID = 928301275168169633L;
+
+    private static final String ENABLED_ELEM = "Enabled";
+    private static final String URL_ELEM = "ConsentUrl";
```

Comment:
this should be something like ConsentPageURL  right?
ConsentUrl doesn't give the correct meaning IMO

Reply:
Resolved by https://github.com/wso2/carbon-identity-framework/pull/4475/commits/22c106db3f58981237cf48cda575f215f1a4a1ff